### DISCUSSION TITLE
Option to skip _patch_bashrc_and_profile

### DISCUSSION THREAD

**Discussion by rvenutolo** [ivan-hc/AM#705] [Ideas]
I would love an option to have the `APP-MANAGER` script skip the function `_patch_bashrc_and_profile` where the script writes the following to `~/.bashrc` (and other dotfiles):

```shell
export PATH=$(echo $PATH | tr ":" "\n" | grep -v "/.local/bin" | tr "\n" ":" | sed s/.$//)
export PATH=$PATH:$HOME/.local/bin
```

Personally, I don't like anything touching my bash dotfiles and prefer to manage my `PATH` manually via my own dotfiles.

Any option to disable that would be great. I don't know if you'd prefer an environment variable to control it like `[[ -z "${APPMAN_SKIP_PATCH_BASHRC_AND_PROFILE:-} ]] && _patch_bashrc_and_profile` or a value in the `appman-config` config file.

---

Unrelated to this, I just want to say I love `appman`. I had ignored AppImages due to the lack of a package management command line tool. Thank you so much for this. This plus `AppImaGen` and `ArchImage` are great. I very much appreciate it.

**Comment by Samueru-sama** [comment]
I turned this into an [issue](https://github.com/ivan-hc/AM/issues/706) with a possible [fix](https://github.com/ivan-hc/AM/pull/707)

Let me know what you think.

**Comment by ivan-hc** [comment]
I would like to move the discussion at https://github.com/ivan-hc/AM/issues/706 , we will have the opportunity to deal with more realities in this regard, given that the "issues" tab is most visited.
